src/state.py

```diff
@@ -214,6 +217,22 @@ def from_env(cls) -> typing.Optional["ProxyConfig"]:
         )
 
 
+def is_storage_ready(charm: ops.CharmBase) -> bool:
+    """Return whether the Jenkins home storage is mounted.
+
+    Args:
+        charm: The Jenkins k8s charm.
+
+    Returns:
+        True if storage is mounted, False otherwise.
+    """
+    container = charm.unit.get_container(JENKINS_SERVICE_NAME)
+    if not container.can_connect():
+        return False
+    mount_info: str = container.pull("/proc/mounts").read()
+    return str(jenkins.HOME_PATH) in mount_info
```

Comment:
Do you think this is a private function? I'm guessing it should only be used to construct the charm state?

Reply:
Fixed! thank you :)